src/module_wrap.cc

```diff
@@ -636,12 +867,28 @@ void ModuleWrap::Initialize(Local<Object> target,
   env->SetProtoMethod(tpl, "instantiate", Instantiate);
   env->SetProtoMethod(tpl, "evaluate", Evaluate);
   env->SetProtoMethod(tpl, "namespace", Namespace);
+  env->SetProtoMethod(tpl, "getStatus", GetStatus);
+  env->SetProtoMethod(tpl, "getDependencySpecifiers", GetDependencySpecifiers);
```

Comment:
maybe rename to getStaticDependencySpecifiers, since this won't pick up `import()`